crates/store/re_grpc_server/src/main.rs

```diff
@@ -0,0 +1,43 @@
+use re_grpc_server::MessageProxy;
+use re_memory::MemoryLimit;
+use re_protos::sdk_comms::v0::message_proxy_server::MessageProxyServer;
+use tokio::net::TcpListener;
+use tonic::transport::server::TcpIncoming;
+use tonic::transport::Server;
+
+#[tokio::main(flavor = "current_thread")]
+async fn main() -> Result<(), tonic::transport::Error> {
+    re_log::setup_logging();
+
+    let tcp_listener = TcpListener::bind("127.0.0.1:1852")
```

Comment:
You should add a `const DEFAULT_GRPC_PORT: u16 = 1852;` somewhere and use it here and elsewhere.

Not only makes it easier to change, but also makes it easy to find by grepping for `grpc.*port`